### DISCUSSION TITLE
在导航栏添加其他菜单结果打不开

### DISCUSSION THREAD

**Discussion by killerss007** [ppoffice/hexo-theme-icarus#1030] [Q&A]
新建了导航栏的菜单，结果点击时一直在网页一直在转圈圈
试过用hexo new page “xxx”新建，还是无法解决
求好心网页解答

**Comment by YiSunYiMao** [comment]
是否忘记在_config.icarus.yml中忘记修改menu配置了？
![图像 2](https://user-images.githubusercontent.com/87130301/153753694-305642bd-5527-4ae5-8bac-3afab4a0cc0a.jpg)
